lib/network_utils.pm

```diff
@@ -151,4 +152,15 @@ sub genmac {
     return lc(join(':', @mac));
 }
 
+=head2 is_networkmanager
+
+ is_networkmanager();
+
+Check if the SUT uses network manager.
+
+=cut
+sub is_networkmanager {
+    return (script_run('readlink /etc/systemd/system/network.service | grep NetworkManager') == 0);
```

Comment:
I kind of hate this, because the network is handled by wicked or NM.
We could check this once, but now we will run the command on each call.

Reply:
Previously it also called this multiple times

Reply:
Yes, I just wanted to change it when I worked on this, but it's not big deal.